packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -81,17 +83,21 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
     // if(isOnboarded()) {
     //   return true;
     // }
-    //get cram_secret from either from AtOnboardingConfig or decode it from qr code whichever available
-    atOnboardingPreference.cramSecret ??=
-        getSecretFromQr(atOnboardingPreference.qrCodePath);
+
+    // get cram_secret from either from AtOnboardingPreference
+    // or fetch from the registrar using verification code sent to email
+    atOnboardingPreference.cramSecret ??= await OnboardingUtil()
+        .getCramUsingOtp(_atSign, atOnboardingPreference.registrarUrl);
```

Comment:
temporarily write an example which tests this method call.

Reply:
change made as part of [5b38eea](https://github.com/atsign-foundation/at_libraries/pull/343/commits/5b38eeaadb5a861431de17602399df43c93ca0e4)